src/main/java/com/codecool/virtualpub/ui/Input.java

```diff
@@ -31,13 +31,14 @@ private String getInput(){
     private int getChoice(Bartender bartender, String[] actions){
         while (true) {
             try {
+                display.displayMessage("\nEnter choice: ");
                 int input = Integer.parseInt(getInput());
-                if (input > actions.length) {
+                if (input > actions.length || input < 1) {
                     throw new Exception();
                 }
                 return input;
             } catch (Exception e) {
-                display.displayMessage("Not a correct choice!");
+                display.displayMessage("\nNot a correct choice!");
```

Comment:
If displayMessage would display in its own line, you wouldn't have to write \n at the beginning of the String